reference-stores/next/components/Header/HeaderMega.js

```diff
@@ -0,0 +1,122 @@
+import { CSSTransition } from 'react-transition-group';
+import Link from 'next/link';
+import Image from 'next/Image';
```

Comment:
```suggestion
import Image from 'next/image';
```

Reply:
Fixed in https://github.com/getnacelle/nacelle-js/pull/133/commits/9d3d0c95de5016f1095d9151bebaddaf07f9e863.